packages/tasks/src/model-libraries.ts

```diff
@@ -524,6 +524,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.kimi_audio,
 		filter: false,
 	},
+	kronos: {
+		prettyLabel: "KRONOS",
+		repoName: "KRONOS",
+		repoUrl: "https://github.com/mahmoodlab/KRONOS",
+		filter: false,
```

Comment:
```suggestion
		filter: false,
		countDownloads: `path_extension:"pt"`,
```

I'd suggest to add a download count rule on the .pt file :)